lstchain/reco/r0_to_dl1.py

```diff
@@ -221,7 +224,10 @@ def r0_to_dl1(
     metadata = global_metadata(source)
     write_metadata(metadata, output_filename)
 
-    cal_mc = load_calibrator_from_config(config)
+    # cal_mc = load_calibrator_from_config(config)
+    from ctapipe.calib import CameraCalibrator
+    # from ctapipe.image.extractor import LocalPeakWindowSum
+    cal_mc = CameraCalibrator(subarray=source.subarray)
```

Comment:
Where is the config for the calibrator read now?

Reply:
Working on it ;-)